gulpfile.js

```diff
@@ -97,6 +98,7 @@ function buildExtensions(options) {
    * Please see {@link AmpCarousel} with `type=slides` attribute instead.
    */
   buildExtension('amp-slides', '0.1', false, options);
+  buildExtension('amp-social-share', '0.1', true, options);
```

Comment:
you'll also need to add your example file near around https://github.com/paul-matthews/amphtml/blob/amp-social-share/gulpfile.js#L326 (please sort)